src/_tests_/Coin.test.js

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from '@testing-library/react';
+import Coin from '../components/Coin';
+import '@testing-library/jest-dom/extend-expect';
+
+const data = {
+  name: 'name',
+  symbol: 'symbol',
+  id: '1',
+  price: '1000',
+};
+describe('coin component', () => {
+  it('elements', () => {
+    render(<Coin
+      name={data.name}
+      symbol={data.symbol}
+      id={data.id}
+      price={data.price}
+      handleDetails={(res) => res}
+    />);
+    expect(screen.getByRole('button')).toBeInTheDocument();
+    expect(screen.getByText('name')).toBeInTheDocument();
+    expect(screen.getByText('symbol')).toBeInTheDocument();
+    expect(screen.getByText(/1000/i)).toBeInTheDocument();
+  });
```

Comment:
- [ ] Nice to test your project using `testing-library/jest-dom` but you still have some of your tests failing so please make sure that you fix these failing tests. :pray: 

![Screenshot from 2023-02-11 18-31-40](https://user-images.githubusercontent.com/92303891/218275532-2064cc6c-15e8-42a7-b988-f0062e702d85.png)